hackclub.com.yaml

```diff
@@ -2074,3 +2074,8 @@ isthackers:
     type: CNAME
     value: isthackers.github.io.
 
+_vercel:
+  ttl: 1
+  type: TXT
+  values:
+    - vc-domain-verify=codethetech.hackclub.com,64d83fc53a66660ef961
```

Comment:
You need to add this to the preexisting `_vercel` and you also need to create a record pointing codethetech to Vercel's CNAME

Reply:
Hey Sam,
I'm adding a GitHub page because it is easier for me to connect rather than connect with vercel.